src/util/PixelBuffer.hxx

```diff
@@ -83,8 +83,12 @@ public:
 
   bool isEmpty() const noexcept;
 
+  uint32_t * data() noexcept;
+  
   static PixelBuffer EMPTY();
 
+  SDL_Surface* toSurface() noexcept;
```

Comment:
Probably need to add a comment here that SDL_Surface* needs to be destroyed before the PixelBuffer is destroyed since they are sharing pixel data